wdl2cwl/tests/oop_cwl_files/bowtie_1.cwl

```diff
@@ -60,25 +60,25 @@ requirements:
         entry: |4
 
             set -e -o pipefail
-            mkdir -p "\$(dirname $(inputs.outputPath ))"
-                    bowtie \
-                    -q \
-                    --sam \
-            $(inputs.seedmms === null ? "" : "--seedmms " inputs.seedmms )\
-            $(inputs.seedlen === null ? "" : "--seedlen " inputs.seedlen )\
-            $(inputs.k === null ? "" : "-k " inputs.k )\
-            $(inputs.best ? "--best" : ""))\
-            $(inputs.strata ? "--strata" : ""))\
-            $(inputs.allowContain ? "--allow-contain" : ""))\
-            $(inputs.threads === null ? "" : "--threads " inputs.threads )\
-            $(inputs.samRG === null ? "" : "--sam-RG '" inputs.samRG )$(inputs.samRG ? "" : "'")\
-            $(inputs.indexFiles[0].replace( "(\.rev)?\.[0-9]\.ebwt$", "") )\
-            $(inputs.readsUpstream.map(function(el) {return el.path}).join(","))\
-                   | picard -Xmx$(inputs.picardXmx) SortSam \
-                   INPUT=/dev/stdin \
-                   OUTPUT=$(inputs.outputPath) \
-                   SORT_ORDER=coordinate \
-                   CREATE_INDEX=true
```

Comment:
Great, the indenting problem is fixed now :-)

Reply:
Yes. Apparently the string has to start from a new line `(\n)` else it can't be handled by textwrap.dedent. So I split the string from the first new line and dedent the right-split and concatenate it to the left-split.